src-common/main/scala/uk/gov/hmrc/play/audit/http/HttpAuditing.scala

```diff
@@ -113,15 +112,21 @@ trait HttpAuditing {
     )
   }
 
-  private def requestDetails(request: HttpRequest)(implicit hc: HeaderCarrier): Map[String, String] =
+  private def when[K, V](pred: Boolean)(value: => Map[K, V]): Map[K, V] =
+    if (pred) value else Map.empty
+
+  private def requestDetails(request: HttpRequest)(implicit hc: HeaderCarrier): Map[String, String] = {
+    val headersMap = request.headers.groupBy(_._1).mapValues(_.map(_._2).mkString(","))
     Map(
-      "ipAddress"            -> hc.forwarded.map(_.value).getOrElse("-"),
-      hc.names.authorisation -> hc.authorization.map(_.value).getOrElse("-"),
-      Path                   -> request.url,
-      Method                 -> request.verb
+      "ipAddress"               -> hc.forwarded.map(_.value).getOrElse("-"),
+      HeaderNames.authorisation -> headersMap.getOrElse(HeaderNames.authorisation, "-"),
+      Path                      -> request.url,
+      Method                    -> request.verb
     ) ++
-      request.body.map(b => Seq(RequestBody -> maskRequestBody(b))).getOrElse(Seq.empty) ++
-      auditExtraHeaderFields(hc.extraHeaders.toMap)
+      headersMap.get("Surrogate").map("surrogate" -> _).toMap ++
+      request.body.map(b => RequestBody -> maskRequestBody(b)).toMap ++
+      when(auditConnector.auditSentHeaders)(headersMap - "Surrogate")
```

Comment:
Dropping `Surrogate` because we add `surrogate` when ever this header is present. Seems a bit pointless having both